Specifications.md

```diff
@@ -1,55 +1,49 @@
-| JAXR (Optional) | [93](https://jcp.org/en/jsr/detail?id=93) | | [GlassFish](https://javaee.github.io/glassfish/) | [GlassFish Issues](https://github.com/javaee/glassfish/issues) |
-| JAX-RPC (Optional) | [101](https://jcp.org/en/jsr/detail?id=101) | | [GlassFish](https://javaee.github.io/glassfish/) | [GlassFish Issues](https://github.com/javaee/glassfish/issues) |
-| Deployment (Optional) | [88](https://jcp.org/en/jsr/detail?id=88) | | [GlassFish](https://javaee.github.io/glassfish/) | [GlassFish Issues](https://github.com/javaee/glassfish/issues) |
+| Technology | Specification | EE4J Implementation |
+|:-----------|:--------------|:--------------------|
+| [Jakarta EE Platform](https://github.com/eclipse-ee4j/jakartaee-platform) | [8](https://jakarta.ee/specifications/platform/8/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta EE Web Profile](https://github.com/eclipse-ee4j/jakartaee-platform) | [8](https://jakarta.ee/specifications/webprofile/8/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Annotations](https://github.com/eclipse-ee4j/common-annotations-api) | [1.3](https://jakarta.ee/specifications/annotations/1.3/) | [Jakarta Annotations](https://eclipse-ee4j.github.io/common-annotations-api/) |
+| [Jakarta Authentication](https://github.com/eclipse-ee4j/jaspic) | [1.1](https://jakarta.ee/specifications/authentication/1.1/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Authorization](https://github.com/eclipse-ee4j/jacc) | [1.5](https://jakarta.ee/specifications/authorization/1.5/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Batch](https://github.com/eclipse-ee4j/batch-api) | [1.0](https://jakarta.ee/specifications/batch/1.0/) |  |
+| [Jakarta Bean Validation](https://github.com/eclipse-ee4j/beanvalidation-spec) | [2.0](https://jakarta.ee/specifications/bean-validation/2.0/) |  |
+| [Jakarta Concurrency](https://github.com/eclipse-ee4j/concurrency-api) | [1.1](https://jakarta.ee/specifications/concurrency/1.1/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Connectors](https://github.com/eclipse-ee4j/jca-api) | [1.7](https://jakarta.ee/specifications/connectors/1.7/) | [Eclipse Glassfish 5.1](https://projects.eclipse.org/projects/ee4j.glassfish/downloads) |
+| [Jakarta Contexts and Dependency Injection](https://github.com/eclipse-ee4j/cdi) | [2.0](https://jakarta.ee/specifications/cdi/2.0/) |  |
+| [Jakarta Debugging Support for Other Languages](https://github.com/eclipse-ee4j/jsp-api) | [1.0](https://jakarta.ee/specifications/debugging/1.0/) | [Eclipse GlassFish 5.1](https://eclipse-ee4j.github.io/glassfish/)  |
```

Comment:
The links for "Jakarta Debugging Support for Other Languages" goes to the JSP API. Is this correct?

Reply:
Yes, the spec is part of that repo:
https://github.com/eclipse-ee4j/jsp-api/tree/master/jdsol-spec